src/main/java/com/matthewglover/tictactoe/core/ComputerPlayer.java

```diff
@@ -3,12 +3,14 @@
 
 public class ComputerPlayer extends Player {
 
+    private static final int MAX_SEARCH_DEPTH = 6;
+
     public ComputerPlayer(PlayerSymbol playerSymbol) {
         super(playerSymbol);
     }
 
     @Override
     protected void makeMove(Game game) {
-        game.move(AlphaBeta.run(game).getMove());
+        game.move(AlphaBeta.run(game, MAX_SEARCH_DEPTH).getMove());
```

Comment:
This use of a static method is problematic. You have two computer strategies, but you've coupled your computer player to using AlphaBeta. Could you perhaps pass the strategy (as an initialized object) into the computer player to make it possible to change implementation? You'd avoid using a static method to boot.